docs/data/external-oasis-repos.md

```diff
@@ -46,7 +46,8 @@ This is the procedure for installing on a Redhat EL7-based system:
 ``` console
 root@host # rpm -i https://repo.grid.iu.edu/osg/3.4/osg-3.4-el7-release-latest.rpm 
 root@host # yum install cvmfs-server.x86_64 osg-oasis 
-root@host # echo "cvmfs_server mount -a" >>/etc/rc.local</pre>
+root@host # echo "cvmfs_server mount -a" >>/etc/rc.local
+root@host # chmod +x /etc/rc.local</pre>
```

Comment:
This looks worse?  Now we're changing the mode of a system file to boot?  The commit message doesn't make it clear why this is necessary and it looks quite strange...

Also, there's an HTML tag left here...

Reply:
I amended the commit message; the comment says if you don't make it executable, it is ignored, so it is necessary to have that chmod.  I removed the html tag and one other.

Reply:
Thanks, much clearer.

However, I'm still a bit nervous about this.  Can you open a ticket to clear it up?  Can either be at the OSG level or with upstream.